CHANGELOG.md

```diff
@@ -25,9 +25,9 @@ Version 7 of the SDK is compatible with Sentry self-hosted versions 24.4.2 or hi
 - Bump JavaScript SDK from v8.54.0 to v9.12.0 ([#4568](https://github.com/getsentry/sentry-react-native/pull/4568), [#4752](https://github.com/getsentry/sentry-react-native/pull/4752))
   - [changelog](https://github.com/getsentry/sentry-javascript/blob/9.12.0/CHANGELOG.md)
   - [diff](https://github.com/getsentry/sentry-javascript/compare/8.54.0...9.12.0)
-- Bump Android SDK from v7.20.1 to v8.8.0 ([#4490](https://github.com/getsentry/sentry-react-native/pull/4490), [#4663](https://github.com/getsentry/sentry-react-native/pull/4663), [#4755](https://github.com/getsentry/sentry-react-native/pull/4755))
-  - [changelog](https://github.com/getsentry/sentry-java/blob/main/CHANGELOG.md#880)
-  - [diff](https://github.com/getsentry/sentry-java/compare/7.20.1...8.8.0)
+- Bump Android SDK from v7.20.1 to v8.9.0 ([#4490](https://github.com/getsentry/sentry-react-native/pull/4490), [#4663](https://github.com/getsentry/sentry-react-native/pull/4663), [#4755](https://github.com/getsentry/sentry-react-native/pull/4755), [#4775](https://github.com/getsentry/sentry-react-native/pull/4775))
```

Comment:
This list is getting quite large, shouldn't we keep the main PR with the main changes and only keep the latest PR that only bumped the version number?